All project code/index.js

```diff
@@ -335,10 +335,37 @@ async function cityToCoordinates(locationInput) {
     // Perform city/country conversion to latitude/longitude coordinates here.
     const country = locationInput.country;
     const city = locationInput.city;
+    var latitude = 0;
+    var longitude = 0;
+    // axios call to API-Ninja geocoding api to return latitude and longitude from city and country input 
+    await axios({
+        // url for API-Ninja
+        url:'https://api.api-ninjas.com/v1/geocoding?city=',
+        method: 'GET',
+        dataType: 'json',
+        headers: {
+            'X-Api-Key': process.env.CITYCOOR_KEY
+        },
+        params: { 
+            'city': city,
+            'country': country
+        }
 
-    let latitude = 0;
-    let longitude = 0;
-
+    }).then(results => {
+        // shows devs if correct call
+        console.log("Successful API call");
+        console.log(JSON.stringify(results.data));
+        //sets lat and long for weather and flight API
+        latitude = JSON.stringify(results.data[0].latitude);
+        longitude = JSON.stringify(results.data[0].longitude);
+        // verifying that lat and long are as I expect 
+        console.log("latitude: ", latitude);
+        console.log("longitude: ", longitude)
+    }).catch(error => {
+        // Handle errors (API call may have failed!)
+        console.log(error);
```

Comment:
Can this error be passed to the user in some way using messages? Check how I made the error code -1 in weather API.

Reply:
If the page simply doesn't load I guess that's fine for now too, the user can always try again. But this should be addressed at some point.